src/components/Page.jsx

```diff
@@ -27,6 +33,18 @@ const Page = () => {
     },
   ];
 
+  const validateFormData = ({activeStep}) => {
+
+    // maybe put a switch statement here to do a different validation for each card?
+    // won't be able to find by Id from another card
+    // should maybe also move the validations to its own component
+    let username = document.getElementById("name").value;
+    username !== '' ? (
+      setFormData({...formData, name: username}),
+      handleNext()
+     ) : setUsernameError(true);
```

Comment:
this setUsernameError isn't working right but idk why. Probably something small and dumb that I'm far in to see